packages/ui/src/molecules/Dropdown/hooks/useDropdownPosition.ts

```diff
@@ -12,15 +12,17 @@ type DropdownPosition = {
  */
 export const useDropdownPosition = (): DropdownPosition => {
   const { dropdownButtonRef } = useDropdown()
+
   const buttonRect = dropdownButtonRef?.current?.getBoundingClientRect()
   const topLevel: number = buttonRect?.top ?? 0
   const topOffset: number = buttonRect?.height ?? 0
-  const topPosition = topLevel + topOffset
-  const leftPosition = buttonRect?.left ?? 0
+  const topPosition = topLevel + topOffset + document.documentElement.scrollTop
+  const leftLevel = buttonRect?.left ?? 0
+  const leftPosition = leftLevel + document.documentElement.scrollLeft
 
   return {
     position: 'absolute',
     top: topPosition,
     left: leftPosition,
-  }
+  } as DropdownPosition
```

Comment:
I can't get enough info just by this context, but is it possible to avoid using this `as`? Why is it needed now?